examples/tracetest-provisioning-env/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+services:
+
+    tracetest:
+        image: kubeshop/tracetest:${TAG:-latest}
+        platform: linux/amd64
+        volumes:
+            - type: bind
+              source: ./tracetest-config.yaml
+              target: /app/tracetest.yaml
```

Comment:
Can we remove this so everything comes from the env variables?

Reply:
Great idea

Reply:
Now that I checked it better, we still have some values that work through the "old" config file, like telemetry settings. this will be changed to a different approach later, so I didn't migrate them.